apps/wallet-dashboard/components/dialogs/staking/views/EnterTimelockedAmountView.tsx

```diff
@@ -68,9 +76,44 @@ export function EnterTimelockedAmountView({
         CoinFormat.FULL,
     );
 
+    const [possibleAmountFormatted, possibleAmountSymbol] = useFormatCoin(
+        possibleAmount,
+        IOTA_TYPE_ARG,
+        CoinFormat.FULL,
+    );
+
     const caption = `${maxTokenFormatted} ${maxTokenFormattedSymbol} Available`;
-    const infoMessage =
-        'It is not possible to combine timelocked objects to stake the entered amount. Please try a different amount.';
+    const info = useMemo(() => {
+        if (isSearchingProtocolMaxAmount) {
+            let message = 'Due to the large number of objects current amount is not valid. ';
```

Comment:
maybe we can use this version: `The current amount is not valid due to the large number of objects. `